pkg/timvir/core/components/Page/components.tsx

```diff
@@ -55,6 +54,8 @@ export const h1 = styled.h1`
   font-size: 36px;
   font-weight: 600;
   text-indent: -0.05em;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Why is this needed?

Reply:
![CleanShot 2024-05-29 at 10 18 57@2x](https://github.com/timvir/timvir/assets/6782188/38e175d7-1f5f-4d9b-b42c-b7e205d0a471)
![CleanShot 2024-05-29 at 10 19 51@2x](https://github.com/timvir/timvir/assets/6782188/cfdd0e0f-d84a-415e-b7c3-a32ac7608338)

Reply:
overflow:hidden causes issues, we can't use that.  i'd prefer word-break:break-word (and tell users to not put long strings without whitespace into the headings.

Reply:
[bac43da](https://github.com/timvir/timvir/pull/2729/commits/bac43daf12cc72f4466b6de0dfa0999e34085704)

Reply:
do you know why the line-height value is missing?
![CleanShot 2024-05-31 at 11 37 18@2x](https://github.com/timvir/timvir/assets/6782188/e8743631-7727-4dc5-aa22-33f794359548)